app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+	package="org.openobservatory.ooniprobe">
+
+	<!-- Allows unlocking your device and activating its screen so UI tests can succeed -->
+	<uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>
+	<uses-permission android:name="android.permission.WAKE_LOCK"/>
+
+	<!-- Allows for storing and retrieving screenshots -->
+	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+	<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+
+	<!-- Allows changing locales -->
+	<uses-permission android:name="android.permission.CHANGE_CONFIGURATION" />
+</manifest>
```

Comment:
Don't know where otherwise to put this comment: what's `v2.db`?

Reply:
the preload database

Reply:
v2 is the name of the database we have on disk, it should have the same name to be overwritten.
https://github.com/ooni/probe-android/blob/master/app/src/main/java/org/openobservatory/ooniprobe/common/AppDatabase.java#L14

Reply:
gotcha!